solana/modules/token_bridge/program/src/api/complete_transfer.rs

```diff
@@ -262,3 +262,21 @@ pub fn complete_wrapped(
 
     Ok(())
 }
+
+/// Verify that the SPL token account `token_account` can receive tokens sent to
+/// `vaa_recipient`.
+/// For this to be the case, one of the following conditions have to hold:
+/// a) `vaa_recipient` matches `token_account`
+/// b) `vaa_recipient` owns `token_account`
```

Comment:
maybe we should strengthen this condition to instead of just an ownership check, check if it's the associated token account. allowing putting it into any owned account might open up some dos attack surface